website_rma/static/src/js/website_rma.js

```diff
@@ -1,7 +1,7 @@
 /* Copyright 2020 Tecnativa - Ernesto Tejeda
 /* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
  */
-odoo.define("website_rma.website_rma", function (require) {
+odoo.define("website_rma.website_rma", function(require) {
```

Comment:
Avoid changing the format as prettier is going to reformat it afterwards and pre-commit will fail.